gradle/libs.versions.toml

```
@@ -10,6 +10,14 @@ kotlin-reflect = { module = "org.jetbrains.kotlin:kotlin-stdlib-reflect", versio
 kotest-junit5-jvm = { module = "io.kotest:kotest-runner-junit5-jvm", version.ref = "kotest" }
 kotest-assertions-core-jvm = { module = "io.kotest:kotest-assertions-core-jvm", version.ref = "kotest" }
 mockito-core = { module = "org.mockito:mockito-core", version.ref = "mockito" }
+jcabi-github = { module = "com.jcabi:jcabi-github", version = "1.2.0" }
+unirest = { module = "com.mashape.unirest:unirest-java", version = "1.4.9" }
+json = { module = "org.json:json", version = "20220320" }
+org-eclipse-jgit = { module = "org.eclipse.jgit:org.eclipse.jgit", version = "6.2.0.202206071550-r" }
+commons-io = { module = "commons-io:commons-io", version = "2.11.0" }
+slf4j-api = { module = "org.slf4j:slf4j-api", version = "2.0.0" }
+logback-classic = { module = "ch.qos.logback:logback-classic", version = "1.3.0-beta0" }
+logback-core = { module = "ch.qos.logback:logback-core", version = "1.3.0-beta0" }
```

Add this in the `[versions] section`
```toml
[versions]
logback = "1.3.0-beta0"
```

and then:

```suggestion
logback-classic = { module = "ch.qos.logback:logback-classic", version = "logback" }
logback-core = { module = "ch.qos.logback:logback-core", version = "logback" }
```